opentracing-spring-cloud-api-extensions/src/main/java/io/opentracing/contrib/spring/cloud/api/extensions/TracerBeanPostProcessor.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright 2017 The OpenTracing Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.opentracing.contrib.spring.cloud.api.extensions;
+
+import java.util.Set;
+import java.util.logging.Logger;
+
+import org.springframework.beans.BeansException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.config.BeanPostProcessor;
+import org.springframework.context.annotation.Configuration;
+
+import io.opentracing.Tracer;
+import io.opentracing.contrib.api.TracerObserver;
+import io.opentracing.contrib.api.tracer.APIExtensionsTracer;
+
+@Configuration
+public class TracerBeanPostProcessor implements BeanPostProcessor {
+
+    private static final Logger log = Logger.getLogger(TracerBeanPostProcessor.class.getName());
```

Comment:
I think we should use what is default/preferred in spring boot, and I guess it's not JUL